### ISSUE TITLE
Connect back and Front end to get a functional website

### ISSUE THREAD

**Issue by janadaouk** [CSAllenISD/2023-ISP-CodeConverter#34]

**Comment by Zaenb** [comment]
I agree with you jana. We should definitely try to do that as soon as possible so that we can see how everything will integrate and whether the front end and back end code will actually work together. We can try to do that starting next week.

**Comment by VivekMaranganti** [comment]
Due to the change and synchronization of the code structure between the multiple Software developers, this process will be a lot easier, as we will be able to connect the input gathering method to the conversion more easily.
